### PR TITLE
Added test for bug report TRUNK-2758. This confirms that the bug is s…

### PR DESCRIPTION
**Pull request by winglam**
…till not fixed and should help developers locate the bug in the source code. https://issues.openmrs.org/browse/TRUNK-2758

### CODE REVIEW

**Comment by dkayiwa** [comment]
@winglam can you include the pull request url on the ticket?

**Comment by winglam** [comment]
I have included the pull request URL on the ticket. Thanks.

**Comment by dkayiwa** [comment]
@winglam can you do the same as for? https://github.com/openmrs/openmrs-core/pull/1780

**Comment by winglam** [comment]
I have ensured that this pull request is consistent with #1780 and updated the ticket with information about this pull request.

File: api/src/test/java/org/openmrs/api/PatientServiceTest.java [hunk 1]
```diff
@@ -920,6 +920,19 @@ public void isIdentifierInUseByAnotherPatient_shouldReturnTrueWhenPatientIdentif
...
+
+	/**
+	 * @see PatientService#isIdentifierInUseByAnotherPatient(PatientIdentifier)
+	 */
+	@Test
+	@Ignore("Unignore after investigating and fixing - TRUNK-2758")
+	@Verifies(value = "should return false when patient identifier Type is Null", method = "isIdentifierInUseByAnotherPatient(PatientIdentifier)")
+	public void
+	isIdentifierInUseByAnotherPatient_shouldReturnFalseWhenPatientIdentifierTypeIsNull()
+			throws Exception {
```
Comment (dkayiwa) [inline api/src/test/java/org/openmrs/api/PatientServiceTest.java]: @winglam shouldn't this test be about creating a patient without an identifier type, as the ticket suggests?

**Comment by dkayiwa** [comment]
Closing because of no response from @winglam

**Pull request closed without merging**